sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -964,6 +964,16 @@ if (contextVersion > 1) {
 	    return gl.getActiveUniforms(program, uniformIndices, pname);
         }
     );
+
+    debug("");
+    debug("Test getUniformBlockIndex");
+    var program = wtu.loadUniformBlockProgram(gl);
+    gl.linkProgram(program);
+    shouldBeTrue('gl.getProgramParameter(program, gl.LINK_STATUS)');
+    shouldBeNonNull('gl.getUniformBlockIndex(program, "Transform")');
+    var blockIndex = gl.getUniformBlockIndex(program, "Transform");
+    gl.uniformBlockBinding(program, blockIndex, 1);
+    shouldBe('gl.getUniformBlockIndex(program, "Transform")', '1');
```

Comment:
Agree with Mo's review. I confirmed that this could be written as:

shouldBe('gl.getActiveUniformBlockParameter(program, ' + blockIndex + ', gl.UNIFORM_BLOCK_BINDING)', '1');

Since this requires some setup it seems acceptable to change this section's name to "Test getUniformBlockIndex and getActiveUniformBlockParameter". I looked through the dEQP tests and this test doesn't seem to be duplicated, so it seems fine to add it here.